### PR TITLE
Uncoddling Prospectors: Part Uno

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
-Reduces Prospector and Salvager pay to zero. Foreman's is left unchanged. -Reduces department start funds from 10600 to 5000.

Go out and make some money.

### CODE REVIEW

**Comment by Rat-Time** [comment]
This is a salt PR asked for in bad faith and adds nothing.

File: code/game/jobs/department.dm [hunk 1]
```diff
@@ -125,7 +125,7 @@
 	name = "Prospectors"
 	id = DEPARTMENT_PROSPECTOR
 	//Nep in 5 hours with full team is 10600
-	account_initial_balance = 10600 //Starts with an even wage to cover payment-expenses. Their job is to make money and, thus, should not start with more money than needed.
+	account_initial_balance = 5000 //Starts with an even wage to cover payment-expenses. Their job is to make money and, thus, should not start with more money than needed.
```
Comment (Trilbyspaceclone) [inline code/game/jobs/department.dm:128]:
```suggestion
	account_initial_balance = 5000 //Starts with money to cover medical issues or getting small gear
```

**Comment by ApplePlastic** [comment]
why

**Comment by AysheDaArtist** [comment]
Oh no.
My 52 credits.
*wipes tears away with 1000 credits found in looting*

Hogg's doesn't pay lazy bones

**Comment by Rat-Time** [comment]
Just more ranting about the 'old days' and making meaningless unwanted changes.

**Comment by AysheDaArtist** [comment]
It's an RP server, history changes, player events change and fuel PRs.
It's canon that the departments are poor right now.
It's lore that Prospectors are required to get their own funds and gear.

This has always and will always be, what makes Sojourn, Sojourn, there is no 'Good ol days' here, it **always has been.**

**Comment by king5327** [comment]
This is an RP server. This kind of shit will just make it tedious.

**Comment by Trilbyspaceclone** [comment]
Host has denied this pr.

**Pull request closed without merging**